### PR TITLE
fix bugzilla Issue 23131 - With -vasm all the X87 instructions are printed wrong

### PR DESCRIPTION
**Pull request by WalterBright**
it was skipping the "fn" prefix.

### CODE REVIEW

**Review by thewilsonator — COMMENTED** [review]
Issue number? test cases?

File: compiler/src/dmd/backend/x86/disasm86.d [hunk 1]
```diff
@@ -1757,14 +1757,16 @@ L2:
     }
 
     puts(p0);
-    put('\t');
-    if (waitflag)
+    put(' ');
+    if (0 && waitflag)
```
Comment (dkorpel) [inline compiler/src/dmd/backend/x86/disasm86.d:1761]: Is there a use for this dead branch?

**Review by dkorpel — APPROVED** [review]

File: compiler/src/dmd/backend/x86/disasm86.d (hunk 1 above)
Comment (WalterBright) [reply compiler/src/dmd/backend/x86/disasm86.d:1761]: Support for the older x87 instruction set.

**Review by RazvanN7 — APPROVED** [review]

**Review by thewilsonator — DISMISSED** [review]
Please retitle the _commit message_ "fix bugzilla Issue 23131..."

**Pull request merged**